packages/backend-common/src/cache/types.ts

```diff
@@ -23,6 +23,12 @@ type ClientOptions = {
    * can be configured per entry at set-time).
    */
   defaultTtl?: number;
+
+  /**
+   * An optional handler for connection errors emitted from the underlying data
+   * store.
+   */
+  onError?: (err: Error) => void;
```

Comment:
Thoughts on having this on the client options vs options for creating the `CacheManager`? I guess with this you can supply a separate error handler for each plugin, but you also have to supply a separate error handler for each plugin :grin:

Reply:
Yeah, this makes sense!  Run-time connection issues with a specific storage engine are probably more a concern for App Integrators, rather than Plugin Developers.  I'll swap this out + provide the default handler that just logs.